examples/CMakeLists.txt

```diff
@@ -0,0 +1,63 @@
+#if(IMGUI_IMPL_ALLEGRO5)
```

Comment:
Remove?

Reply:
put them in there as a todo, I don't have the setups or patience to fix it for all the examples.

Reply:
> to fix it for all the examples

Do we need examples there? Maybe just leave an `imgui` library?

Reply:
we dont need them but thought I would offer to changes upstream